pkg/cmd/cli/cmd/request_project.go

```diff
@@ -52,7 +52,7 @@ func NewCmdRequestProject(name, fullName, ocLoginName, ocProjectName string, f *
 	cmd := &cobra.Command{
 		Use:     fmt.Sprintf("%s NAME [--display-name=DISPLAYNAME] [--description=DESCRIPTION]", name),
 		Short:   "Request a new project",
-		Long:    fmt.Sprintf(requestProjectLong, ocLoginName, ocProjectName),
+		Long:    requestProjectLong,
```

Comment:
@deads2k Interestingly enough this does have adverse affects and the change fixes it ... here's what I see today:
```
$ oc new-project -h
Create a new project for yourself

Assuming your cluster admin has granted you permission, this command will create a new project
for you and assign you as the project admin. If your administrator has not given you permission to
create your own projects, contact your system administrator.

After your project is created it will be made your default project in your config.%!(EXTRA string=oc login, string=oc project)
```